### PR TITLE
BUG: set link as inappropriate if contains string in ignore_if_contains

### PR DESCRIPTION
**Pull request by lsternlicht**

### CODE REVIEW

File: instapy/like_util.py [hunk 1]
```diff
@@ -505,7 +505,7 @@ def check_link(browser,
         return True, user_name, is_video, 'Username'
 
     if any((word in image_text for word in ignore_if_contains)):
-        return False, user_name, is_video, 'None'
+        return True, user_name, is_video, 'None'
```
Comment (lsternlicht) [inline instapy/like_util.py:508]: If the link contains a word in ignore list, then we want to mark it as inappropriate (i.e. return True)

**Comment by sionking** [comment]
ignore_if_contains is to ignore the set_dont_like if a word exist in line for example:
line is : "sexy girl do laundry"
set_dont_like = sex
ignore_if_contains = laundry

than this line is OK

**Comment by sionking** [comment]
I do agree name shall be changed.

**Comment by timgrossmann** [comment]
@lsternlicht Thank you very much!
Good thing you spotted this mistake.

Looking forward to future contributions!

**Pull request merged**